guides/common/modules/con_upgrade-paths.adoc

```diff
@@ -14,7 +14,6 @@ endif::[]
 ifndef::foreman-deb[]
 . Upgrade your {ProjectServer}:
 .. Upgrade your {ProjectServer} to {ProjectVersion}.
-.. Optional: Upgrade the operating system on your {ProjectServer} to {EL}{nbsp}9.
```

Comment:
The steps on this line and line 29 actually included a note, you should probably drop the notes too.

Reply:
Added as a separate commit. Please squash when merging.